src/db/OrganizationSchema.ts

```diff
@@ -0,0 +1,70 @@
+import mongoose from 'mongoose'
+import muuid from 'uuid-mongodb'
+
+import { OrganizationType, OrgType, IOrganizationContent, OperationType } from './OrganizationTypes.js'
+import { ChangeRecordMetadataType } from './ChangeLogType.js'
+
+const { Schema, connection } = mongoose
+
+const ChangeRecordMetadata = new Schema<ChangeRecordMetadataType>({
+  user: {
+    type: 'object',
+    value: { type: 'Buffer' },
+    required: true
+  },
+  historyId: { type: Schema.Types.ObjectId, ref: 'change_logs' },
+  prevHistoryId: { type: Schema.Types.ObjectId, ref: 'change_logs' },
```

Comment:
I thinking about retiring this field.  Initially I thought it'd be useful to have both current and previous history id so we can build a simple query to show previous change. However, when doing bulk updates (using bulkWrite) it's difficult to get the current historyId to assign to prevHistoryId.